README.md

```diff
@@ -12,3 +12,13 @@ $ google-chrome-stable --remote-debugging-port=9899
 In a python console, you can connect to the remote debugging port and enable the Page domain.
 
 > devtools_client = ChromeInterface(9899, domains={"Page": {}})
+
+# Contributing
```

Comment:
There is no content in "contributing"? It look like this PR is actually adding how to run the tests.

Reply:
Yes, there's nothing much to put, so I just thought to put this. Should I close this request ?